libs/stack/stack-ui/src/components/Alerts/components/AlertsNavigationButton.tsx

```diff
@@ -1,20 +1,18 @@
 import type { Ref } from 'react'
 import { forwardRef } from 'react'
 import { FocusRing } from 'react-aria'
-import useThemeContext from '../../../providers/Theme/hooks'
+import { ButtonWithForwardRef } from '../../Button'
 import type { TButtonProps } from '../../Button/interface'
 import Icon from '../../Icon'
 
 const AlertsNavigationButton = forwardRef((props: TButtonProps, ref: Ref<HTMLButtonElement & HTMLAnchorElement>) => {
   const { themeName = 'alerts.navigationBtn', tokens, customTheme, children, ...rest } = props
 
-  const theme = useThemeContext(themeName, tokens, customTheme)
-
   return (
     <FocusRing within focusRingClass="has-focus-ring">
-      <button className={theme} ref={ref} type="button" {...rest}>
+      <ButtonWithForwardRef themeName={themeName} tokens={tokens} ref={ref} {...rest}>
```

Comment:
Utiliser le componsant Button plutot que l'Élément button brise le onKeyDown (for some reason, j'imagine que c'est relié a react swiper)

Voir https://github.com/OKAMca/cepsum-mono/blob/CEP-156-Brand-ui-Alerts/libs/ui/src/components/Alerts/components/AlertsNextButton.tsx